xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -35,6 +35,8 @@
 #include <string>
 #include <vector>
 
+using namespace std;
+
```

Comment:
We shouldn't do this in public API headers - it dumps a load of symbols into the global namespace for user code.

Explicitly qualify with `std::` instead.

Reply:
fixed in a fixup commit.